ci/eval/default.nix

```diff
@@ -66,6 +67,7 @@ let
             "$src/ci/eval/attrpaths.nix" \
             -A paths \
             -I "$src" \
+            --argstr extraNixpkgsConfig "${builtins.toJSON extraNixpkgsConfig}" \
```

Comment:
Here we can either pass `env.extraNixpkgsConfig` to `runCommand` to get a shell var, or we can use `lib.escapeShellArg` or `lib.toShellVars` to handle escaping the string.

Just wrapping it in quotes manually is not enough.